src/containers/BlogTemplate.js

```diff
@@ -2,46 +2,33 @@ import * as React from 'react';
 import Grid from '@mui/material/Grid';
 import MainFeaturedPost from './MainFeaturedPost';
 import FeaturedPost from './FeaturedPost';
+import BubblesBackground from "../images/bubbles.png";
+import GameConsoles from '../images/gameconsoles.png';
+import GameGraphic from '../images/gamegraphic.png';
 
 const mainFeaturedPost = {
-    title: 'PartyFish',
-    description: 'A free and open source party game for up to 10 players',
-    image: 'https://images.unsplash.com/photo-1464925257126-6450e871c667?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=774&q=80',
+    title: 'PartyFishGames',
```

Comment:
I thought we were going by PartyFish and partyfishgames was just for github because partyfish was already taken haha, will bring this up with team

Reply:
Oops - you're right. I just changed it back to PartyFish.